edb/edgeql/ast.py

```diff
@@ -385,6 +385,8 @@ class IfElse(Expr):
     condition: Expr
     if_expr: Expr
     else_expr: Expr
+    # Just affects pretty-printing
+    python_style: bool = False
```

Comment:
We probably don't need this as we can always codegen the new syntax - semantically they are equivalent and we can pick a preferred opinionated syntax.